docs/reference/release-notes/highlights-7.2.0.asciidoc

```diff
@@ -4,14 +4,12 @@
 <titleabbrev>7.2.0</titleabbrev>
 ++++
 
-coming[7.2.0]
-
 //NOTE: The notable-highlights tagged regions are re-used in the
 //Installation and Upgrade Guide
 
 // tag::notable-highlights[]
 [discrete]
-==== {dataframes-cap}
+==== {dataframes-cap} (Beta)
```

Comment:
I don't think you need the "Beta" in both the title and the first paragraph. In Kibana they used the latter only and that's my preference too.

Reply:
`(Beta)` removed in https://github.com/elastic/elasticsearch/pull/43343/commits/d8d8e594568a44c15162c00ec23f1befaf403fc0